internal/functions/efgs/api/structs.go

```diff
@@ -117,11 +117,16 @@ type DiagnosisKeyWrapper struct {
 
 //ToData convert struct from DiagnosisKeyWrapper to DiagnosisKey
 func (wrappedKey *DiagnosisKeyWrapper) ToData() *DiagnosisKey {
+	rt := wrappedKey.TransmissionRiskLevel
+	if rt > 8 || rt < 0 {
+		rt = 0 // This is an override for putting 0x7fffffff there
+	}
+
```

Comment:
why is this needed? can we document it a bit better in the code?

Reply:
This is a sad story of how EFGS has put this into their protocol and then recommended to not use it because the value meaning can be different across different countries. Because it _cannot_ be unused in fact, the final recommendation is to put max value possible there. That works for others but, because we put those keys into the Key server, not for us - the Key server doesn't like it so we have to adjust the value to basically anything _consumable_. The value itself is not used at all for anything and even though it's said to be _optional_ in comments, the Key server requires it and fails when the value is not there or is invalid.  
Sad story.

Reply:
I can surely put this novel above into the code if you like @marekaf

Reply:
yes please 👍